### ISSUE TITLE
Why not use the mean of the loss?

### ISSUE THREAD

**Issue by auzxb** [barronalex/Tacotron#5]
Why not use the mean  of loss like this？(https://github.com/barronalex/Tacotron/blob/master/models/tacotron.py#L136)

        seq2seq_loss = tf.reduce_mean(tf.abs(seq2seq_output - mel))
        output_loss = tf.reduce_mean(tf.abs(output - linear))
